macOS/LocalPackages/NewTabPage/Sources/NewTabPage/PrivacyStats/NewTabPagePrivacyStatsClient.swift

```diff
@@ -28,11 +28,8 @@ public final class NewTabPagePrivacyStatsClient: NewTabPageUserScriptClient {
     private var cancellables: Set<AnyCancellable> = []
 
     enum MessageName: String, CaseIterable {
-        case getConfig = "stats_getConfig"
         case getData = "stats_getData"
-        case onConfigUpdate = "stats_onConfigUpdate"
         case onDataUpdate = "stats_onDataUpdate"
-        case setConfig = "stats_setConfig"
```

Comment:
Config messages for Privacy Stats and Recent Activity are removed because they were only managing visibility, which is now handled in the top-level Protections widget.